scripts/__input_class_player/__input_class_player.gml

```diff
@@ -323,6 +329,10 @@ function __input_class_player() constructor
         if ((__rebind_state > 0) && (array_length(__source_array) > 0)) __binding_scan_failure(INPUT_BINDING_SCAN_EVENT.SOURCE_CHANGED);
         
         __color_set();
+        
+        var _gamepad = __source_get_gamepad();
+        if (_gamepad > -1) __input_gamepad_stop_trigger_effects(_gamepad);
+        
```

Comment:
Do we also need to stop trigger effects when removing a source? (i.e. in the `__source_remove()` method)